tests/FakeItEasy.Tests/FakeObjectCallExtensionsTests.cs

```diff
@@ -156,38 +137,6 @@ public void WriteToConsole_should_be_null_guarded()
             call.Should().BeNullGuarded();
         }
 
-        [Fact]
-        public void WriteToConsole_should_call_writer_registered_in_container_with_calls()
-        {
-            // Arrange
-            var calls = A.CollectionOfFake<IFakeObjectCall>(2);
-
-            var callWriter = A.Fake<CallWriter>();
-            this.StubResolve(callWriter);
-
-            // Act
-            calls.WriteToConsole();
-
-            // Assert
-            A.CallTo(() => callWriter.WriteCalls(calls, A<IOutputWriter>._)).MustHaveHappened();
-        }
-
-        [Fact]
-        public void WriteToConsole_should_call_writer_registered_in_container_with_console_out()
-        {
-            // Arrange
-            var calls = A.CollectionOfFake<IFakeObjectCall>(2);
-
-            var callWriter = A.Fake<CallWriter>();
-            this.StubResolve(callWriter);
-
-            // Act
-            calls.WriteToConsole();
-
-            // Assert
-            A.CallTo(() => callWriter.WriteCalls(A<IEnumerable<IFakeObjectCall>>._, A<IOutputWriter>._)).MustHaveHappened();
-        }

```

Comment:
Yeah, just removed. No replacement for these tests, unlike all the others.

Also, @thomaslevesque has suggested removing `WriteToConsole`. I'd even consider removing `Write`.